# Color sequence submission

# Color sequence
string color1
string color2
string color3

---
bool success
